### PR TITLE
Transfer all duplicates.

### PR DESCRIPTION
**Pull request by necrozis**
Short Description: 
with --firsttrans key you will transfer all your duplicated pokemons, except the most CP.
If your pokemon dont have twins skip it.

### CODE REVIEW

File: pokemongo_bot/__init__.py [hunk 1]
```diff
@@ -160,10 +112,50 @@ def _setup_api(self):
...
+        inventory_dict = inventory_req['responses']['GET_INVENTORY']['inventory_delta']['inventory_items']
+       
+        print('[x] First Transfer.')
+        
+        for pokemon in inventory_dict:
+            try:
+                id1 = pokemon['inventory_item_data']['pokemon_data']['pokemon_id']
+                id2 = pokemon['inventory_item_data']['pokemon_data']['id']
+                id3 = pokemon['inventory_item_data']['pokemon_data']['cp']
+
```
Comment (douglascamata) [inline pokemongo_bot/__init__.py]: consider renaming id{1,2,3} to something more meaning, like `pokemon_id`, `id`, and `cp`

**Comment by douglascamata** [comment]
Can you also please break this function into smaller ones for better understanding? Because a mistake here can be expensive.

**Comment by necrozis** [comment]
@douglascamata Refactored. Check please.

**Comment by douglascamata** [comment]
`This branch has conflicts that must be resolved` :(

**Comment by necrozis** [comment]
@douglascamata just need to remove 86-140 lines.

File: pokemongo_bot/__init__.py [hunk 2]
```diff
@@ -160,10 +112,55 @@ def _setup_api(self):
...
 
+        if self.config.firsttrans:
+            self.first_transfer()
+            
+        print('[#]')
         self.update_inventory();
-
+        
+    def first_transfer_get_groups(self):
+        self.pokemon_groups = {}
```
Comment (douglascamata) [inline pokemongo_bot/__init__.py]: @necrozis you mind accumulating in `pokemon_groups`(without `self`) and then returning it in the end? Also please add an `_` to the beginning of the name of this method (it indicates it's a private method) and move it below the `first_transfer` method.

**Comment by douglascamata** [comment]
@necrozis and you got conflicts again, dude. LOL

**Comment by necrozis** [comment]
@douglascamata i just try to undestang how githubs works ahaha sorry xDD

**Comment by necrozis** [comment]
wow

**Comment by douglascamata** [comment]
@necrozis looks good. Did you test?

**Comment by necrozis** [comment]
@douglascamata fixed, tested on my main acc :dash:

**Comment by douglascamata** [comment]
@necrozis one last thing, I promise: add information about this in the README.

**Comment by necrozis** [comment]
@douglascamata Done.

**Pull request merged**